src/pages/SignUp.vue

```diff
@@ -6,23 +6,23 @@
         <div class="input-group">
           <label class="control-label col-sm-4" for="name">Name</label><i class="bar"></i>
           <div class="col-sm-8">
-            <input type="text" id="name" required="required"/>
+            <input type="text" id="name" required="required" v-model="nickname"/>
```

Comment:
What does the nickname refer to

Reply:
User will provide a name that we use to address them.